airbyte-cdk/python/airbyte_cdk/sources/file_based/file_types/avro_parser.py

```diff
@@ -146,8 +146,8 @@ def parse_records(
             raise ValueError(f"Expected ParquetFormat, got {avro_format}")
 
         line_no = 0
-        try:
-            with stream_reader.open_file(file, self.file_read_mode, self.ENCODING, logger) as fp:
+        with stream_reader.open_file(file, self.file_read_mode, self.ENCODING, logger) as fp:
```

Comment:
If I understand this correctly, the problem is that the wide `catch Exception` catches everything, and we only wanted the problems related to parsing the record itself?